[2020-12-22 13:57:29,965] {taskinstance.py:655} INFO - Dependencies all met for <TaskInstance: loan_prediction_train_model.preprocess_data 2020-12-22T12:43:23.210460+00:00 [queued]>
[2020-12-22 13:57:29,980] {taskinstance.py:655} INFO - Dependencies all met for <TaskInstance: loan_prediction_train_model.preprocess_data 2020-12-22T12:43:23.210460+00:00 [queued]>
[2020-12-22 13:57:29,980] {taskinstance.py:866} INFO - 
--------------------------------------------------------------------------------
[2020-12-22 13:57:29,980] {taskinstance.py:867} INFO - Starting attempt 32 of 36
[2020-12-22 13:57:29,980] {taskinstance.py:868} INFO - 
--------------------------------------------------------------------------------
[2020-12-22 13:57:29,993] {taskinstance.py:887} INFO - Executing <Task(LivyBatchOperator): preprocess_data> on 2020-12-22T12:43:23.210460+00:00
[2020-12-22 13:57:29,998] {standard_task_runner.py:53} INFO - Started process 52640 to run task
[2020-12-22 13:57:30,058] {logging_mixin.py:112} INFO - Running %s on host %s <TaskInstance: loan_prediction_train_model.preprocess_data 2020-12-22T12:43:23.210460+00:00 [running]> dcb43f18a24b
[2020-12-22 13:57:30,083] {logging_mixin.py:112} INFO - [2020-12-22 13:57:30,083] {batch.py:214} INFO - Submitting the batch to Livy... Payload:
{
  "file": "s3a://alanchn31-loan-default-prediction/dist/main.py",
  "args": [
    "--job",
    "preprocess_data",
    "--phase",
    "train",
    "--mode",
    "aws",
    "--awsKey",
    "AKIAUW4FKHTMR2UR3INH",
    "--awsSecretKey",
    "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",
    "--s3Bucket",
    "alanchn31-loan-default-prediction"
  ],
  "pyFiles": [
    "s3a://alanchn31-loan-default-prediction/dist/src.zip"
  ],
  "name": "preprocess_data_scheduled__2020-12-22T12:43:23.210460+00:00"
}
[2020-12-22 13:57:30,092] {logging_mixin.py:112} INFO - [2020-12-22 13:57:30,091] {base_hook.py:84} INFO - Using connection to: id: livy. Host: http://ec2-3-236-26-7.compute-1.amazonaws.com, Port: 8998, Schema: None, Login: None, Password: None, extra: None
[2020-12-22 13:57:30,095] {logging_mixin.py:112} INFO - [2020-12-22 13:57:30,095] {http_hook.py:136} INFO - Sending 'POST' to url: http://ec2-3-236-26-7.compute-1.amazonaws.com:8998/batches
[2020-12-22 13:57:30,124] {logging_mixin.py:112} INFO - [2020-12-22 13:57:30,124] {batch.py:166} INFO - Batch successfully submitted with id = 28.
[2020-12-22 13:57:30,125] {logging_mixin.py:112} INFO - [2020-12-22 13:57:30,125] {batch.py:80} INFO - Getting batch 28 status...
[2020-12-22 13:57:30,133] {logging_mixin.py:112} INFO - [2020-12-22 13:57:30,133] {base_hook.py:84} INFO - Using connection to: id: livy. Host: http://ec2-3-236-26-7.compute-1.amazonaws.com, Port: 8998, Schema: None, Login: None, Password: None, extra: None
[2020-12-22 13:57:30,136] {logging_mixin.py:112} INFO - [2020-12-22 13:57:30,136] {http_hook.py:136} INFO - Sending 'GET' to url: http://ec2-3-236-26-7.compute-1.amazonaws.com:8998/batches/28
[2020-12-22 13:57:30,145] {logging_mixin.py:112} INFO - [2020-12-22 13:57:30,145] {batch.py:90} INFO - Batch 28 has not finished yet (state is 'starting')
[2020-12-22 13:57:50,163] {logging_mixin.py:112} INFO - [2020-12-22 13:57:50,163] {batch.py:80} INFO - Getting batch 28 status...
[2020-12-22 13:57:50,171] {logging_mixin.py:112} INFO - [2020-12-22 13:57:50,170] {base_hook.py:84} INFO - Using connection to: id: livy. Host: http://ec2-3-236-26-7.compute-1.amazonaws.com, Port: 8998, Schema: None, Login: None, Password: None, extra: None
[2020-12-22 13:57:50,173] {logging_mixin.py:112} INFO - [2020-12-22 13:57:50,173] {http_hook.py:136} INFO - Sending 'GET' to url: http://ec2-3-236-26-7.compute-1.amazonaws.com:8998/batches/28
[2020-12-22 13:57:50,182] {logging_mixin.py:112} INFO - [2020-12-22 13:57:50,182] {batch.py:299} INFO - --------------------------------------------------Full log for batch 28--------------------------------------------------
[2020-12-22 13:57:50,189] {logging_mixin.py:112} INFO - [2020-12-22 13:57:50,189] {base_hook.py:84} INFO - Using connection to: id: livy. Host: http://ec2-3-236-26-7.compute-1.amazonaws.com, Port: 8998, Schema: None, Login: None, Password: None, extra: None
[2020-12-22 13:57:50,192] {logging_mixin.py:112} INFO - [2020-12-22 13:57:50,192] {http_hook.py:136} INFO - Sending 'GET' to url: http://ec2-3-236-26-7.compute-1.amazonaws.com:8998/batches/28/log?from=0&size=100
[2020-12-22 13:57:50,199] {logging_mixin.py:112} INFO - [2020-12-22 13:57:50,199] {batch.py:309} INFO - stdout: 
[2020-12-22 13:57:50,201] {logging_mixin.py:112} INFO - [2020-12-22 13:57:50,201] {batch.py:309} INFO - 2020-12-22 13:57:31,918 WARN util.NativeCodeLoader: Unable to load native-hadoop library for your platform... using builtin-java classes where applicable
[2020-12-22 13:57:50,203] {logging_mixin.py:112} INFO - [2020-12-22 13:57:50,203] {batch.py:309} INFO - Traceback (most recent call last):
[2020-12-22 13:57:50,205] {logging_mixin.py:112} INFO - [2020-12-22 13:57:50,205] {batch.py:309} INFO -   File "/tmp/spark-c2a06914-c4a8-4d7c-bd9b-3efdfdf5042f/main.py", line 74, in <module>
[2020-12-22 13:57:50,205] {logging_mixin.py:112} INFO - [2020-12-22 13:57:50,205] {batch.py:309} INFO -     main()
[2020-12-22 13:57:50,205] {logging_mixin.py:112} INFO - [2020-12-22 13:57:50,205] {batch.py:309} INFO -   File "/tmp/spark-c2a06914-c4a8-4d7c-bd9b-3efdfdf5042f/main.py", line 55, in main
[2020-12-22 13:57:50,205] {logging_mixin.py:112} INFO - [2020-12-22 13:57:50,205] {batch.py:309} INFO -     with open("src/config.json", "r") as config_file:
[2020-12-22 13:57:50,207] {logging_mixin.py:112} INFO - [2020-12-22 13:57:50,207] {batch.py:309} INFO - FileNotFoundError: [Errno 2] No such file or directory: 'src/config.json'
[2020-12-22 13:57:50,207] {logging_mixin.py:112} INFO - [2020-12-22 13:57:50,207] {batch.py:309} INFO - 2020-12-22 13:57:34,842 INFO util.ShutdownHookManager: Shutdown hook called
[2020-12-22 13:57:50,208] {logging_mixin.py:112} INFO - [2020-12-22 13:57:50,208] {batch.py:309} INFO - 2020-12-22 13:57:34,843 INFO util.ShutdownHookManager: Deleting directory /tmp/spark-c2a06914-c4a8-4d7c-bd9b-3efdfdf5042f
[2020-12-22 13:57:50,208] {logging_mixin.py:112} INFO - [2020-12-22 13:57:50,208] {batch.py:309} INFO - 2020-12-22 13:57:34,850 INFO impl.MetricsSystemImpl: Stopping s3a-file-system metrics system...
[2020-12-22 13:57:50,208] {logging_mixin.py:112} INFO - [2020-12-22 13:57:50,208] {batch.py:309} INFO - 2020-12-22 13:57:34,850 INFO impl.MetricsSystemImpl: s3a-file-system metrics system stopped.
[2020-12-22 13:57:50,208] {logging_mixin.py:112} INFO - [2020-12-22 13:57:50,208] {batch.py:309} INFO - 2020-12-22 13:57:34,850 INFO impl.MetricsSystemImpl: s3a-file-system metrics system shutdown complete.
[2020-12-22 13:57:50,208] {logging_mixin.py:112} INFO - [2020-12-22 13:57:50,208] {batch.py:309} INFO - 
stderr: 
[2020-12-22 13:57:50,208] {logging_mixin.py:112} INFO - [2020-12-22 13:57:50,208] {batch.py:309} INFO - 
YARN Diagnostics: 
[2020-12-22 13:57:50,209] {logging_mixin.py:112} INFO - [2020-12-22 13:57:50,208] {batch.py:309} INFO - spark-submit start failed
[2020-12-22 13:57:50,209] {logging_mixin.py:112} INFO - [2020-12-22 13:57:50,209] {batch.py:318} INFO - --------------------------------------------------End of full log for batch 28--------------------------------------------------
[2020-12-22 13:57:50,209] {logging_mixin.py:112} INFO - [2020-12-22 13:57:50,209] {batch.py:335} INFO - Closing batch with id = 28
[2020-12-22 13:57:50,219] {logging_mixin.py:112} INFO - [2020-12-22 13:57:50,219] {base_hook.py:84} INFO - Using connection to: id: livy. Host: http://ec2-3-236-26-7.compute-1.amazonaws.com, Port: 8998, Schema: None, Login: None, Password: None, extra: None
[2020-12-22 13:57:50,221] {logging_mixin.py:112} INFO - [2020-12-22 13:57:50,221] {http_hook.py:136} INFO - Sending 'DELETE' to url: http://ec2-3-236-26-7.compute-1.amazonaws.com:8998/batches/28
[2020-12-22 13:57:50,229] {logging_mixin.py:112} INFO - [2020-12-22 13:57:50,229] {batch.py:340} INFO - Batch 28 has been closed
[2020-12-22 13:57:50,238] {taskinstance.py:1128} ERROR - Batch 28 failed with state 'dead'
Traceback (most recent call last):
  File "/opt/bitnami/python/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 966, in _run_raw_task
    result = task_copy.execute(context=context)
  File "/usr/local/airflow/plugins/airflow_livy/batch.py", line 173, in execute
    ).execute(context)
  File "/opt/bitnami/python/lib/python3.6/site-packages/airflow/sensors/base_sensor_operator.py", line 107, in execute
    while not self.poke(context):
  File "/usr/local/airflow/plugins/airflow_livy/batch.py", line 96, in poke
    raise AirflowException(f"Batch {self.batch_id} failed with state '{state}'")
airflow.exceptions.AirflowException: Batch 28 failed with state 'dead'
[2020-12-22 13:57:50,241] {taskinstance.py:1151} INFO - Marking task as UP_FOR_RETRY
[2020-12-22 13:57:55,006] {logging_mixin.py:112} INFO - [2020-12-22 13:57:55,006] {local_task_job.py:103} INFO - Task exited with return code 1
